src/main/java/com/faforever/client/remote/FafServerAccessor.java

```diff
@@ -101,6 +108,12 @@ public void start() {
                                  .retry()
                                  .subscribe();
 
+      getEvents(VetoesChangedInfo.class).publishOn(fxApplicationThreadExecutor.asScheduler())
```

Comment:
Since this is team matchmaking specific the team matchmaking service should do the subscribing and have the onVetoesChanged function.

This aalso resolves the circular dependency